Readme.md

```diff
@@ -20,8 +20,8 @@ Source code: https://github.com/geodynamics/burnman
 
 Forums: https://community.geodynamics.org/c/burnman
 
-Authors (as of 2015, listed alphabetically by first name):
-* Bob Myhill
+Authors (as of 2021):
+* Bob (Robert) Myhill (main contributer)
```

Comment:
```suggestion
* Bob (Robert) Myhill
```
I'm actually ok without the extra flag, but thank you for thinking of this :)

If we're removing the "listed alphabetically by first name" convention, should we adopt the same order as the author list in conf.py?

Reply:
I thought some variation is fine, as long as you are first. I think the extra flag is needed. Almost went ahead and added your email. It would be good if it was clear to people whom to approach if they did have suggestions or wanted to get involved.

Reply:
Ok, sure, I'm ok with that. In which case, there's a typo to fix ;)